assessments/projects/openfga/self-assessment.md

```diff
@@ -1,7 +1,5 @@
 # OpenFGA Security Self Assessment
 
-This assessment was created by community members as part of the [Security Pals](https://github.com/cncf/tag-security/issues/1102) process and is currently pending changes from the maintainer team.

```

Comment:
Would it be okay to change this so it acknowledges the Security Pals effort, but indicates the maintainers have completed the process?

Reply:
Totally.